vars.yml

```diff
@@ -0,0 +1,3 @@
+prom_user: "{{ ( prom_secret.stdout_lines[0] | b64decode | split(':'))[0] }}"
```

Comment:
Where are ``prom_secret`` and ``prom_route`` being set?

Reply:
in e2e.yml

Reply:
Unless you use ``include_vars: vars.yml`` in some task after getting the ``prom_secret``, the ``prom_secret`` won't be available.

Reply:
I am including vars.yml when I run the playbook -e @vars.yml